gradle.properties

```diff
@@ -13,7 +13,7 @@ org.gradle.jvmargs=-Xmx1536m
 org.gradle.parallel=true
 
 GROUP=com.amazonaws
-VERSION_NAME=3.1.3
+VERSION_NAME=3.1.4
```

Comment:
Don't we usually have to decrement the version of the gradle plugin when we do a release?  Can you just confirm that this change can be built manually first?

Reply:
I'm manually overriding the plugin version for now until we figure out a better way. The gradle plugin hasn't changed anyways.